tests/integration/test_interaction.py

```diff
@@ -407,14 +475,35 @@ def test_keyboard_help(self):
         self.interact_with_keyboard_help(use_keyboard=True)
 
 
+@ddt
+class KeyboardAssessmentInteractionTest(BasicAssessmentInteractionTest, BaseIntegrationTest):
```

Comment:
@itsjeyd In the last commit I removed the entire `KeyboardAssessmentInteractionTest`, but I also removed `BaseIntegrationTest` from other classes that already pulled that class in via mixins or parent classes.